codeship/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Exit script with error if any step fails.
+set -e
+
+go test -v ./monitor/
```

Comment:
For future growth, maybe just test everything.
```suggestion
go test -v ./...
```

Reply:
Done